src/components/DataCollector/createEvent.js

```diff
@@ -29,6 +29,9 @@ export default () => {
     mergeDevice: createMerger(content, "device"),
     mergeEnvironment: createMerger(content, "environment"),
     mergePlaceContext: createMerger(content, "placeContext"),
+    expectsResponse() {
+      return Boolean(content.query);
```

Comment:
My only comment:

I think we should we assume that we expect a response if the event type is `viewStart`, even if query is `undefined`. Thoughts?

Other than that, looks great.

Reply:
I talked this over with @jfkhoury with our mouth-holes. We agree that we should move forward with the way it is here, but we'll be discussing `query` in more depth with Rares on Monday and will go from there.